frontend/src/components/additionalActions/index.js

```diff
@@ -0,0 +1,67 @@
+
+import { Grid, SvgIcon, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import { observer } from 'mobx-react'
+import Widget, { WidgetTitle } from '../widget'
+import { ReactComponent as RobotStandIcon } from './robot_stand.svg'
+import { ReactComponent as RobotSitIcon } from './robot_sit.svg'
+
+const RobotPositionButton = observer(({heading = '', position, currentPosition, children}) => {
+  const theme = useTheme()
+  const isActive = position === currentPosition
+
+  return <Grid container item sx={{
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: isActive ? theme.palette.info.main : theme.palette.background.robotPositionButton,
+    width: '80px',
+    height: '80px',
+    borderRadius: '26px',
+    margin: '0 12px',
+  }}>
+    {children}
+    <Typography
+      sx={{
+        fontSize: '11px',
+        color: isActive ? theme.palette.common.white : theme.palette.background.robotPositionTitle,
+        fontWeight: theme.typography.fontWeightRegular,
+      }}
+
+    >{heading}</Typography>
+  </Grid>
+})
+
+
+const AdditionalActions = observer(() => {
+  const theme = useTheme()
+  // TODO: define logic for additional actions in a store (data, requests to DB, etc.)
+  const robotPosition = 'sit' // or 'stand'
```

Comment:
Can we make positions - an enum? To avoid string duplication...

Reply:
done